src/components/Reviews/SingularReview.jsx

```diff
@@ -17,6 +18,32 @@ export const SingularReview = () => {
       setIsLoading(false);
     });
   }, []);
+
+  const upVote = (review_id) => {
```

Comment:
Would be great to notify the user if this upvote fails. Consider having a state that keeps track of if there has been an error, then conditionally render an error message.

Love the optimistic rendering, and the votes are persistent which is good.